docs/compass/08-01-basic-flow.md

```diff
@@ -0,0 +1,101 @@
+
+3. In the **APIs** tab, copy the URL to your Application. You will use it in the Compass UI in the next steps.
+
+### Register your Application in the Compass UI
+
+1. Open a separate tab in your browser and go to `https://compass.{CLUSTER_DOMAIN}`. It will navigate you to the Compass UI. From the drop-down list on the top navigation panel, select a tenant you want to work on. For the purpose of this tutorial, select the `default` tenant. In the **Runtimes** tab, there is already the default `kymaruntime` that you can work on to complete this tutorial. Make sure that your Runtime is assigned to the `DEFAULT` scenario.
+
+2. In the left navigation panel, navigate to the **Application** tab and click **Create application...** to register your Application in Compass. Choose **From scratch** from the drop-down list. For the purpose of this tutorial, name your Application `test-app`. By default, your Application is assigned to the `DEFAULT` scenario.
+
+3. Select `test-app` in the **Applications** view and add the API spec of the Order Service:
+
+    a. Click the **+** button in the **Packages** section and fill in all the required fields. For the purpose of this tutorial, name your Package `test-package`.
+
+    b. In the **Credentials** tab, choose `None` from the drop-down list to select the credentials type. For the purpose of this tutorial, there is no need to secure the connection. Click **Create**.
+
+    c. Navigate to the `test-package` Package and click the **+** button in the **API Definitions** section. Fill in all the required fields. In the **Target URL** field, paste the URL to your Application.
+
+    d. Click the **Add specification** button and upload the `order-service` [API spec file](./assets/order-service-api-spec.yaml). Click **Create**.
+
+
+### Use your Application in the Kyma Console UI
+
+1. Go back to the Kyma Console UI. You can see that the `test-app` Application is registered in the **Applications** view. Select `test-app` and bind it to your Namespace by selecting the **Create Binding** button.
+
+2. From the drop-down list in the top-right corner, select your Namespace and go to the **Catalog** view. You will see your services available under the **Services** tab. Provision the service instance by choosing your Package and clicking the **Add** button in the top-right corner of the page.
+
+3. Create a lambda function. In the **Overview** tab, click the **Deploy new resource** button and upload the file with the [lambda function](./assets/lambda.yaml).
+
+4. Expose your lambda:
+
+    a. In the left navigation panel, go to the **Lambdas** tab and click the `call-order-service` lambda.
+
+    b. In the **Settings & Code** section, click the **Select Function Trigger** button and expose your lambda via HTTPS.
+
+    c. Untick the **Enable authentication** field as there is no need to secure the connection for the purpose of this tutorial. Click **Add**.
```

Comment:
```suggestion
    c. Untick the **Enable authentication** option as there is no need to secure the connection for the purpose of this tutorial. Click **Add**.
```
I feel like an `option` or a `box` would go better with `untick`. You can't really untick a field, can you?